migrations/20260617222400-profile-cms-pointer-events.js

```
@@ -1,14 +1,23 @@
 const fs = require('fs');
 const path = require('path');
 
+function readSqlStatements(filePath) {
+  return fs
+    .readFileSync(filePath, { encoding: 'utf8' })
+    .split(';')
+    .map((statement) => statement.trim())
+    .filter((statement) => statement.length > 0);
+}
+
 exports.up = async function (db) {
```

ESLint

[error] 'exports' is not defined.

(no-undef)